src/pebble.py

```diff
@@ -401,6 +419,11 @@ def reconcile(  # noqa: C901 pylint: disable=too-many-branches,too-many-statemen
             ignore_order=True,
             ignore_string_case=True,
         )
+
+        restart_mas(container, rendered_mas_configuration)
```

Comment:
To be set before restart_syname with a comment explaning it must be done before synapse